app_feup/lib/view/Widgets/copy_center_card.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'generic_card.dart';
+import 'package:uni/view/Widgets/info_desk_card.dart';
+
+class CopyCenterCard extends GenericCard {
+  CopyCenterCard({Key key}) : super(key: key);
+
+  CopyCenterCard.fromEditingInformation(
+      Key key, bool editingMode, Function onDelete)
+      : super.fromEditingInformation(key, editingMode, onDelete);
```

Comment:
I'm not sure this function is really necessary. Since this card can't be added to the main page, it can't really be edited by the user. I'm not aware of any plans to make these cards editable in the future, but if that's not the case this function can be removed.
The same goes for all the cards on this page.
(Bear in mind that I'm only just starting to use Flutter, so if this comment doesn't make sense feel free to just ignore it :upside_down_face: )

Reply:
For now, we aren't planning to make this page editable. So, in fact, that function seems useless. 
I'm not very acquainted with Flutter too, but I think we can remove it. When we started working on this issue we copied another card from the main page and that's probably the reason why those lines of code exist... We've never deleted them. Let's do it now :)